tests/test_integration_argon_metad.py

```
@@ -0,0 +1,233 @@
+        height=0.05,  # Small height in eV
+        pace=10,  # Deposit Gaussian every 10 steps
+        temp=120.0,  # Temperature in Kelvin (Argon melts at ~84 K)
+        biasfactor=10.0,  # Well-tempered metadynamics
+        file="HILLS",
+        flush=10
+    )
+
+    # Step 5: Create minimal model class that provides the LJ calculator
+    class LJModel:
+        """Minimal model for testing that wraps LJ calculator."""
+        def get_calculator(self, directory=None, **kwargs):
```

Ruff (0.14.4)

Unused method argument: `directory`

(ARG002)